<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kasiski</class>
 <widget class="QWidget" name="kasiski">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontal_layout_1">
     <item>
      <widget class="QTabWidget" name="tab_widget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_text">
        <attribute name="title">
         <string>Text</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="group_box_input">
           <property name="title">
            <string>Input text</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextEdit" name="text_edit_input"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_document">
        <attribute name="title">
         <string>Document</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_box_stats">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string>Statistics</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="text_edit_stats"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="vertical_layout_1">
       <item>
        <widget class="QGroupBox" name="group_box_options">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="form_layout_options">
            <item row="0" column="0">
             <widget class="QLabel" name="label_seq_len">
              <property name="text">
               <string>Sequence length</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spin_box_seq_len">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_threshold">
              <property name="text">
               <string>Threshold</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spin_box_threshold">
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_min_key_length">
              <property name="text">
               <string>Min key length</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_max_key_length">
              <property name="text">
               <string>Max key length</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spin_box_min_key_length">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spin_box_max_key_length">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vertical_spacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontal_spacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vertical_layout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="button_analysis">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Analysis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vertical_spacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vertical_spacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
